src/spellcast.py

```diff
@@ -98,12 +103,25 @@ def legal_moves(self, sort_key = None, sort_reverse: bool = True):
         legal_move_nodes: list[SearchNode] = []
 
         # Record all legal moves from all root tiles on the board
-        for y in range(len(self.tiles)):
-            for x in range(len(self.tiles[y])):
-                if TileModifier.FROZEN in self.tile_at(x, y).modifiers:
-                    continue
-                
-                legal_move_nodes += self.legal_moves_from(x, y)
+        if config['multiProcessing']:
```

Comment:
Please can we use double quotes, as has been used throughout the rest of the codebase

Reply:
Sure, changed